app/screens/channel/channel.ios.js

```diff
@@ -65,10 +64,6 @@ export default class ChannelIOS extends ChannelBase {
                             valueEvent={CHANNEL_POST_TEXTBOX_VALUE_CHANGE}
                         />
                     </View>
-                    <ChannelLoader
```

Comment:
we can remove the entire channel_loader file if we want to go ahead and remove that loader screen